raiden/ui/app.py

```diff
@@ -270,7 +270,9 @@ def run_app(
             click.secho(
                 f"'{pathfinding_service_info['message']}'. "
                 f"You have chosen pathfinding operator '{pathfinding_service_info['operator']}' "
-                f"with the running version '{pathfinding_service_info['version']}' ",
+                f"with the running version '{pathfinding_service_info['version']}' "
+                f"with chain_id:'{pathfinding_service_info['network_info']['chain_id']}."
```

Comment:
I'd write "on chain_id"

Reply:
There's a space missing after the period.

Reply:
thanks done